source/content/pantheon-yml.md

```diff
@@ -117,6 +115,42 @@ php_version: 7.0
 * From time to time, we will roll out a new default version of PHP, which will be available to apply as a one-click update in the Dashboard. If you are overriding the default, make sure to remove `php_version` from `pantheon.yml` as soon as possible to ensure you don't miss the latest recommended PHP version.
 * You'll always be able to test new default PHP version in Dev and Test before deploying Live.
 
+### Specify a Version of MariaDB
+
+<Alert type="info" title="Do not set the database version unless the site is eligible.">
+
+The site is eligible to upgrade if the site's Dashboard shows a blue banner across the top that reads:
+
+> Good news, your site's database version is now configurable! Learn how.
+
+</Alert>
+
+Keep the software your site uses current and up to date, or set a specific version to avoid incompatibilities.
+
+Use the `database` directive in `pantheon.yml` to choose a specific version of MariaDB:
+
+```yaml:title=pantheon.yml
+database:
+  version: 10.4
+```
+
+Keep in mind that some versions of Drupal and WordPress require a specific minimum or maximum version for compatibility.
+
+Not all CMS versions can be configured to use a specific database version on Pantheon yet.
+
+This table shows the recommended MariaDB version for each CMS, as well as an estimate of when the database can be configured:
+
+| CMS            | Recommended MariaDB Version | Configurable After |
+|----------------|-----------------------------|--------------------|
+| Drupal <6.51   | Default                     | May, 2021          |
+| Drupal >= 6.51 | Default                     | May, 2021          |
+| Drupal < 7.76  | Default                     | May, 2021          |
+| Drupal >= 7.76 | Default                     | May, 2021          |
+| Drupal < 8.5   | Default                     | May, 2021          |
+| Drupal >= 8.6  | 10.0 - 10.4                 |                    |
+| Drupal >= 9.0  | 10.4 - 10.4                 |                    |
```

Comment:
```suggestion
| Drupal >= 8.6  | 10.0 or 10.4                |                    |
| Drupal >= 9.0  | 10.4                        |                    |
```
Specifying a range like `10.0 - 10.4` makes it sound like you can select any version inside that range. Instead, you can right now only choose `10.0` or `10.4`. So we should make that clear.

Also, probably no need to specify a range on Drupal 9.

Reply:
looks like @joa-pan fixed this one - marking resolved